.github/workflows/memory-leak-testing.yml

```diff
@@ -39,6 +39,9 @@ jobs:
           steps.cache-pnpm-dependencies.outputs.cache-hit != 'true' ||
           steps.cache-pnpm-packages.outputs.cache-hit != 'true'
 
+      - name: Install Puppeteer browser
+        run: npx puppeteer browsers install chrome
```

Comment:
memlab already uses this under the hood, remove it!

Reply:
I was looking for the reason why the tests weren't running. The error message indicated that this module was missing, as you can see in the logs.